packages/windows.treeinfo.json

```diff
@@ -1,7 +1,6 @@
 {
   "variants": {
     "adminrouter": "windows",
-    "dcos-metrics": "windows",
```

Comment:
Telegraf ostensibly supports windows, so we should be able to get it to build just by adding a package variant for windows. Getting it configured to run on windows is another matter. I'm not certain that `dcos-metrics` is being used on windows since we don't have any integration tests covering it. If it is then we'll need to update CI with an integration test that we can use to determine whether we've successfully replaced it.